src/current/cockroachcloud/pci-dss.md

```diff
@@ -5,9 +5,9 @@ toc: true
 docs_area: manage.security
 ---
 
-{{ site.data.products.dedicated }} has been certified by a PCI Qualified Security Assessor (QSA) as a PCI DSS [Level 1 Service Provider](https://www.pcidssguide.com/what-are-pci-service-provider-compliance-levels/). This certification extends the existing [SOC 2 Type 2 certification](https://www.cockroachlabs.com/blog/soc-2-compliance-2/) of {{ site.data.products.dedicated }}, which provides a baseline level of security controls to safeguard customer data.
+{{ site.data.products.dedicated }} advanced has been certified by a PCI Qualified Security Assessor (QSA) as a PCI DSS [Level 1 Service Provider](https://www.pcidssguide.com/what-are-pci-service-provider-compliance-levels/). This certification extends the existing [SOC 2 Type 2 certification](https://www.cockroachlabs.com/blog/soc-2-compliance-2/) of {{ site.data.products.dedicated }}, which provides a baseline level of security controls to safeguard customer data.
```

Comment:
```suggestion
{{ site.data.products.dedicated }} has been certified by a PCI Qualified Security Assessor (QSA) as a PCI DSS [Level 1 Service Provider](https://www.pcidssguide.com/what-are-pci-service-provider-compliance-levels/). This certification extends the existing [SOC 2 Type 2 certification](https://www.cockroachlabs.com/blog/soc-2-compliance-2/) of {{ site.data.products.dedicated }}, which provides a baseline level of security controls to safeguard customer data.
```

Reply:
The auditor didn't audit "Dedicated Advanced" as a specific capabilities. They audited the "dedicated" offering as a whole. Advanced is a pricing plan within that offering so let's not make any changes in the PCI context.

Reply:
I reverted changes to this file except for line 108.